src/browser/provider/pool.js

```diff
@@ -39,11 +42,13 @@ export default {
             browserName  = providerRegExpMatch[1] || '';
         }
 
-        return { provider, providerName, browserName };
+        browserOption = provider.plugin.getConfig(browserName);
+
+        return { provider, providerName, browserName, browserOption };
     },
 
     async _parseAlias (alias) {
-        if (alias.browserName && alias.providerName && alias.provider)
+        if (Object(alias) === alias && BROWSER_INFO_PROPERTIES.every(property => property in alias))
```

Comment:
The problem is that some properties may be empty. So we need to change all property in the initial condition with `in`. If the alias again is changed we will need to change the condition again and it will become too long maybe.